computation_graph/run.py

```diff
@@ -59,13 +60,19 @@ def _make_computation_input(args, kwargs):
     return base_types.ComputationInput(args=args, kwargs=kwargs)
 
 
-_get_node_ambiguous_edge_groups = toolz.compose_left(
+_incoming_edge_options = gamla.compose_left(
     graph.get_incoming_edges_for_node,
-    curried.groupby(lambda edge: edge.key),
-    curried.valmap(toolz.compose_left(curried.sorted(key=lambda edge: edge.priority))),
-    dict.values,
-    gamla.star(itertools.product),
-    curried.map(tuple),
+    gamla.after(
```

Comment:
is this to allow currying for `get_incoming_edges_for_node` ?

Reply:
exactly